HomeWork19/Task3.py

```diff
@@ -0,0 +1,6 @@
+class MyStr(str):
+    def __init__(self, text: str):
+        self.text = text
+
+    def __str__(self):
+        return self.text.upper()
```

Comment:
nit: could be simplified
```suggestion
    def __str__(self):
        return self.upper()
```